ci-operator/step-registry/upi/install/azure/upi-install-azure-commands.sh

```diff
@@ -0,0 +1,297 @@
+AZURE_AUTH_LOCATION="${CLUSTER_PROFILE_DIR}/osServicePrincipal.json"
+export AZURE_AUTH_LOCATION
+
+pushd ${ARTIFACT_DIR}/installer
+
+CLUSTER_NAME=$(python3 -c 'import yaml;data = yaml.full_load(open("install-config.yaml"));print(data["metadata"]["name"])')
+BASE_DOMAIN=$(python3 -c 'import yaml;data = yaml.full_load(open("install-config.yaml"));print(data["baseDomain"])')
+AZURE_REGION=$(python3 -c 'import yaml;data = yaml.full_load(open("install-config.yaml"));print(data["platform"]["azure"]["region"])')
+BASE_DOMAIN_RESOURCE_GROUP=$(python3 -c 'import yaml;data = yaml.full_load(open("install-config.yaml"));print(data["platform"]["azure"]["baseDomainResourceGroupName"])')
+SSH_PUB_KEY=$(<"${CLUSTER_PROFILE_DIR}/ssh-publickey")
+export CLUSTER_NAME
+export BASE_DOMAIN
+
+date "+%F %X" > "${SHARED_DIR}/CLUSTER_INSTALL_START_TIME"
+echo "Creating manifests"
+openshift-install --dir=${ARTIFACT_DIR}/installer create manifests
+
+echo "Editing manifests"
+sed -i '/^  channel:/d' manifests/cvo-overrides.yaml
+rm -f openshift/99_openshift-cluster-api_master-machines-*.yaml
+rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
+sed -i "s;mastersSchedulable: true;mastersSchedulable: false;g" manifests/cluster-scheduler-02-config.yml
+sed -i "/publicZone/,+1d" manifests/cluster-dns-02-config.yml
+sed -i "/privateZone/,+1d" manifests/cluster-dns-02-config.yml
+
+popd
+
+echo "Creating ignition configs"
+openshift-install --dir=${ARTIFACT_DIR}/installer create ignition-configs &
+wait "$!"
+
+cp ${ARTIFACT_DIR}/installer/bootstrap.ign ${SHARED_DIR}
+BOOTSTRAP_URI="https://${JOB_NAME_SAFE}-bootstrap-exporter-${NAMESPACE}.svc.ci.openshift.org/bootstrap.ign"
+export BOOTSTRAP_URI
+# begin bootstrapping
+
+mkdir -p /tmp/azure
+
+# Copy sample UPI files
+cp -r /var/lib/openshift-install/upi/azure/* /tmp/azure
+
+echo "az version:"
+az version
+
+echo "Logging in with az"
+AZURE_AUTH_CLIENT_ID=$(cat $AZURE_AUTH_LOCATION | jq -r .clientId)
+AZURE_AUTH_CLIENT_SECRET=$(cat $AZURE_AUTH_LOCATION | jq -r .clientSecret)
+AZURE_AUTH_TENANT_ID=$(cat $AZURE_AUTH_LOCATION | jq -r .tenantId)
+az login --service-principal -u $AZURE_AUTH_CLIENT_ID -p "$AZURE_AUTH_CLIENT_SECRET" --tenant $AZURE_AUTH_TENANT_ID
+
+INFRA_ID="$(jq -r .infraID ${ARTIFACT_DIR}/installer/metadata.json)"
+RESOURCE_GROUP="${INFRA_ID}-rg"
+echo "Infra ID: ${INFRA_ID}"
+
+echo "Creating resource group ${RESOURCE_GROUP}"
+az group create --name $RESOURCE_GROUP --location $AZURE_REGION
+
+echo "Creating identity"
+az identity create -g $RESOURCE_GROUP -n ${INFRA_ID}-identity
+
+ACCOUNT_NAME=$(echo ${CLUSTER_NAME}sa | tr -cd '[:alnum:]')
+
+echo "Creating storage account"
+az storage account create -g $RESOURCE_GROUP --location $AZURE_REGION --name $ACCOUNT_NAME --kind Storage --sku Standard_LRS
+ACCOUNT_KEY=$(az storage account keys list -g $RESOURCE_GROUP --account-name $ACCOUNT_NAME --query "[0].value" -o tsv)
+
+VHD_URL="$(cat /var/lib/openshift-install/rhcos.json | jq -r .azure.url)"
```

Comment:
This needs to use the output from `openshift-install coreos print-stream-json` if exists, like we did for the other platforms.